array_api_tests/test_creation_functions.py

```diff
@@ -119,7 +136,7 @@ def test_full(shape, fill_value, dtype):
 
     if dtype is None:
         # TODO: Should it actually match the fill_value?
-        # assert a.dtype in _floating_dtypes, "eye() should produce an array with the default floating point dtype"
+        # assert a.dtype in _floating_dtypes, "eye() should returned an array with the default floating point dtype"
```

Comment:
```suggestion
        # assert a.dtype in _floating_dtypes, "eye() should return an array with the default floating point dtype"
```

Reply:
As long as you are cleaning up the other tests in this file you might as well implement this.

Reply:
Oops I forgot about this, will do.